sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerModelFactory.cs

```diff
@@ -272,11 +288,28 @@ public static FieldValue FieldValueWithDictionaryValueType(IReadOnlyDictionary<s
         /// <param name="text">The text of this form element.</param>
         /// <param name="words">A list of the words that make up the line.</param>
         /// <returns>A new <see cref="FormRecognizer.Models.FormLine"/> instance for mocking.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public static FormLine FormLine(FieldBoundingBox boundingBox, int pageNumber, string text, IReadOnlyList<FormWord> words)
         {
             words = words?.ToList();
 
-            return new FormLine(boundingBox, pageNumber, text, words);
+            return new FormLine(boundingBox, pageNumber, text, words, default);
+        }
```

Comment:
Regarding these old methods we're hiding, would it make more sense to make them call the newer overload?
```cs
[EditorBrowsable(EditorBrowsableState.Never)]
public static FormLine FormLine(FieldBoundingBox boundingBox, int pageNumber, string text, IReadOnlyList<FormWord> words) =>
  FormLine(boundingBox, pageNumber, text, words, default);
```

This way we can keep a single method in the factory with an actual implementation, which avoids repeating code.